### ISSUE TITLE
plugin markdown

### ISSUE THREAD

**Issue by l337quez** [geany/geany-plugins#757]
the markdown plugin seems to be out of date, I wanted to place a box with python code and the code will be highlighted, but in egeany with the plugin is not highlighted. I want to know if it is a problem in the installation of the plugin or that it is actually outdated. In case of being outdated, I want to help. how can I do it

**Comment by elextr** [comment]
You could try applying #747 and see if it works?

**Comment by codebrainz** [comment]
I don't believe either the [original](https://daringfireball.net/projects/markdown/syntax) or [cmark](http://commonmark.org/) spec discusses syntax highlighting at all. If you can get it added to CommonMark, then it will "just work" in the Markdown plugin.

**Comment by l337quez** [comment]
@codebrainz @elextr  I saw that I had not installed cmark, I installed it, but the error persisted I saw a commonmark in the repositories of ARCH but it says Arch Linux - python-commonmark .. I do not know what I should do to make it work

**Comment by codebrainz** [comment]
> ...but the error persisted...

What error?

To be clear, syntax highlighting of code blocks is **NOT** part of "standard" Markdown. If you get it added to the [CommonMark specification](https://spec.commonmark.org/), that is, by working with the [CommonMark group](http://commonmark.org/) and convincing them to make it a standard feature, once it's in the standard and implemented, it will just work in the Geany Markdown plugin. Until then, installing patches or extra packages won't help.

**Comment by l337quez** [comment]
thanks

**Comment by codebrainz** [comment]
@laoshaw a bit off topic for this issue, but Markdown is still part of Geany-Plugins as always. If it's missing from your distribution packages, you should file a bug report or use a more stable distro.
